RELEASE-NOTES.txt

```diff
@@ -19,6 +19,7 @@
 - Fix ol numbering in markdown preview [#1823](https://github.com/Automattic/simplenote-electron/pull/1823)
 - Prevents weird effects in live previews due to incomplete input [#1822](https://github.com/Automattic/simplenote-electron/pull/1822)
 - Fixed a bug where searching for a tag containing non-alphanumeric characters erroneously returned no notes [#1828](https://github.com/Automattic/simplenote-electron/pull/1828)
+- Properly close revision/history view when clicking outside of slider [#1837](https://github.com/Automattic/simplenote-electron/pull/1837)
```

Comment:
Is this part of 1.14.0 or 1.14.0?

Reply:
I don't know. We haven't released 1.14.0 - if you are saying we have to create 1.14.1 I can do that, but it seemed like this is just one part of the release process so I didn't think we needed to create a new release.

Reply:
@dmsnell @belcherj I _think_ we can still call this 1.14.0 since there wasn't a public release yet.

/cc @loremattei